src/main/flight/mixer.h

```diff
@@ -115,6 +115,8 @@ extern int16_t motor_disarmed[MAX_SUPPORTED_MOTORS];
 struct motorConfig_s;
 struct rxConfig_s;
 
+uint8_t getMotorCount();
```

Comment:
We normally don't user getters / setters in this project. Just add `uint8_t motorCount` to make it accessible.